pkg/reconcilermanager/controllers/reposync_controller_test.go

```diff
@@ -1594,13 +1612,17 @@ func TestRepoSyncUpdateOverrideAPIServerTimeout(t *testing.T) {
 		setServiceAccountName(nsReconcilerName),
 		secretMutator(nsReconcilerName+"-"+reposyncSSHKey),
 		containerEnvMutator(repoContainerEnv),
+		setResourceVersion("2"),
 	)
 	updatedRepoDeployment.ResourceVersion = "2"
 	wantDeployments[core.IDOf(repoDeployment)] = updatedRepoDeployment
 
 	if err := validateDeployments(wantDeployments, fakeDynamicClient); err != nil {
 		t.Errorf("Deployment validation failed. err: %v", err)
 	}
+	if t.Failed() {
```

Comment:
nit: can replace `Errorf` with `Fatalf` to fail now

ditto elsewhere

Reply:
This whole file is a little inconsistent about that. Sometimes it uses Fatal and sometimes it uses Error and fails later.

I think the goal was to show multiple errors when there was one, perhaps to help debug after a Reconcile. So I tried to make sure there was a Fail if Failed before every Log printing success. Some have multiple tests after the Reconcile and some one have one. So I left the check the same for all of them.